tests/unit_test/data_processor/upload_handler_log_processor_test.py

```
@@ -0,0 +1,144 @@
+        Utility.load_environment()
+        connect(**Utility.mongoengine_connection(Utility.environment['database']["url"]))
+
+    def test_add_log_creates_new(self):
+        bot = 'test_bot'
+        user = 'test_user'
+        collection_name = 'test_collection'
+        file_name="Salesstore.csv"
+        upload_type="crud_data"
+        UploadHandlerLogProcessor.add_log(bot, user,  file_name=file_name, upload_type=upload_type, collection_name=collection_name,)
+        log = UploadHandlerLogs.objects(bot=bot).get().to_mongo().to_dict()
+        assert not log.get('exception')
```

Ruff (0.12.2)

Use of `assert` detected

(S101)